build.gradle

```diff
@@ -62,7 +62,11 @@ buildscript {
 }
 
 plugins {
-    id "io.gitlab.arturbosch.detekt" version "1.0.0.RC6-3"
+    id "io.gitlab.arturbosch.detekt" version "1.14.0"
+}
+
+repositories {
+    jcenter()
```

Comment:
@sleepylee do we still need this config as we had it in project gradle already?

Reply:
I followed their instruction to have a top-level `repositories`🤔  maybe we can try to remove it 
![image](https://user-images.githubusercontent.com/13435717/94769247-c99a8400-03db-11eb-976e-f5cbf3a54a13.png)

Reply:
@sleepylee so, will you remote it? 🤔

Reply:
yes we can remove it 💯 confirmed and will commit soon